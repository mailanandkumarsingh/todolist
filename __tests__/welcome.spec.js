import React from 'react';
import Welcome from '../src/views/welcome';
import renderer from 'react-test-renderer';

describe('Welcome', () => {
  it('renders Welcome coponent correctly', () => {
    const tree = renderer.create(
      <Welcome name="World" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
