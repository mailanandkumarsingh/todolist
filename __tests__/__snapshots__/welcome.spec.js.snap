// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders Welcome coponent correctly 1`] = `
<h1>
  Hello to you, 
  World
</h1>
`;
